modules/ROOT/pages/configure-studio-to-use-your-own-maven.adoc

```diff
@@ -0,0 +1,24 @@
+= Configure Studio to Use Your Own Maven
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Anypoint Studio comes bundled with Maven but you can use Studio with an external Maven installation as well.
+
+. Navigate to Studio preferences:
```

Comment:
Make sure this is consistent with "open studio and go to preferences" steps in other topics.